tools/codeowners-linter/Azure.Sdk.Tools.CodeownersLinter/Azure.Sdk.Tools.CodeownersLinter.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
As we talked at some point we should merge codeowners parser and this linter so that we don't have this logic duplicated as we shouldn't need both.